src/singletons/Settings.hpp

```diff
@@ -528,6 +528,10 @@ class Settings : public ABSettings, public ConcurrentSettings
          {"d", 1},
          {"w", 1}}};
 
+    BoolSetting pluginsEnabled = {"/plugins/supportEnabled", false};
+    ChatterinoSetting<std::vector<QString>> enabledPlugins = {
```

Comment:
Could this be a `ChatterinoSetting<std::set<QString>>`? This would help the `PluginController::isPluginEnabled` method (and its callers) since it wouldn't involve checking all plugins.

Reply:
It would be simpler but there is no implementation for ChatterinoSetting<std::set<...>>